sphinx_js/typedoc.py

```diff
@@ -0,0 +1,277 @@
+import os
+import sys
+import json
+
+# JSDoc JSON schema: https://github.com/jsdoc3/jsdoc/blob/master/lib/jsdoc/schema.js
+# JSDoc entries used in sphinx-js:
```

Comment:
Fantastic piece of documentation. We should promote this to somewhere more conspicuous.